package.json

```diff
@@ -16,9 +16,6 @@
     "codegen": "gql-gen --config codegen.yml",
     "audit": "./scripts/audit/audit.sh"
   },
-  "jest": {
```

Comment:
Was causing a warning from Jest that that is was ignoring this as there is a `jest.config.js` (it's not needed also as the GitHub Actions workflow specifies `--testResultsProcessor`).